build.gradle

```diff
@@ -57,38 +58,68 @@ dependencies {
     implementation libs.nva.json
     testImplementation libs.bundles.testing
 }
+tasks.named('test', Test) {
+    useJUnitPlatform()
+    failFast = false
+    testLogging {
+        events = ['skipped', 'passed', 'failed']
+        showCauses = true
+        exceptionFormat = "full"
+    }
+}
 
-testing {
-    suites {
-        // Configure the built-in test suite
-        test {
-            // Use JUnit4 test framework
-            useJUnit('4.13.2')
+tasks.named("jacocoTestReport", JacocoReport) {
+    dependsOn(tasks.named('test'))
+    enabled = true
+    reports {
+        xml.required = true
+        xml.outputLocation = layout.buildDirectory.file("reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml").get().asFile
```

Comment:
better ways to do this @LarsV123 ? pipeline need the report inside testCodeCoverageReport folder

Reply:
This is fine for now I think. This is a custom path set by a plugin we don't use/need here (`jacoco-report-aggregation`). Should probably fix the pipeline to handle custom paths.